src/customizations/volto/components/theme/Navigation/Navigation.jsx

```diff
@@ -98,7 +98,9 @@ const Navigation = ({ pathname }) => {
           <HeaderToggler
             aria-controls="it-navigation-collapse"
             aria-expanded={collapseOpen}
-            aria-label="Toggle navigation"
+            aria-label={intl.formatMessage(messages.ToggleMenu, {
+              action: collapseOpen ? 'Chiudi' : 'Apri',
```

Comment:
Aggiustare anche qui le traduzioni come nella pr per v3 (far in modo che siano effettivamente aggiunte a locales E tradurre anche Chiudi e Apri)

Reply:
Fatto